### PR TITLE
Add all subclass Levels and features

### LINKED ISSUE #176: Some subclasses missing from Levels
Currently, we only include the subclasses for Bard and Barbarian. We're missing the SRD subclasses for everything else.

I'm not sure if we're missing these subclasses in other places as well. It's probably worth checking out.

### PR DESCRIPTION
**Pull request by fergcb**
## What does this do?
- Removes `features` array from Subclass objects, as features are given by subclass levels
- Adds all subclass levels from the SRD as Level objects, containing feature lists

## How was it tested?
JSON validated with ESLint, tested with Jest.

## Is there a Github issue this is resolving?
Resolves #176 

## Here's a fun image for your troubles
defect or feature

### CODE REVIEW

File: src/5e-SRD-Levels.json [hunk 1]
```diff
@@ -8158,5 +8154,974 @@
...
+    "class": {
+      "name": "Cleric",
+      "url": "/api/classes/cleric"
+    },
+    "subclass": {
+      "name": "Life",
+      "url": "/api/subclasses/life"
+    },
+    "url": "/api/subclasses/life/levels/1",
+    "index": 248
```
Comment (bagelbits) [inline src/5e-SRD-Levels.json:8182]: Not for this PR, but we might as well swap out the index for `${class.name || subclass.name}-${level}`

File: src/5e-SRD-Levels.json [hunk 2]
```diff
@@ -8158,5 +8154,974 @@
...
     "url": "/api/subclasses/lore/levels/14",
     "index": 247
+  },
+  {
+    "level": 1,
+    "feature_choices": [],
+    "features": [
+      {
+        "name": "Bonus Proficiency",
+        "url": "/api/features/bonus-proficiency"
```
Comment (bagelbits) [inline src/5e-SRD-Levels.json:8164]: I hope this doesn't bite us in the ass later. But it's fine for now.

File: src/5e-SRD-Levels.json [hunk 3]
```diff
@@ -8158,5 +8154,974 @@
...
+    "url": "/api/subclasses/land/levels/2",
+    "index": 253
+  },
+  {
+    "level": 6,
+    "feature_choices": [],
+    "features": [
+      {
+        "name": "Land's Stride",
+        "url": "/api/features/lands-stride"
```
Comment (bagelbits) [inline src/5e-SRD-Levels.json]: This URL doesn't exist. I believe you want `/api/features/druid-lands-stride`

Comment (bagelbits) [reply src/5e-SRD-Levels.json]: You may want to do a once over and make sure all of your feature urls exist. I wish the CI covered this check but it doesn't yet.

File: src/5e-SRD-Levels.json (hunk 1 above)
Comment (fergcb) [reply src/5e-SRD-Levels.json:8182]: Agreed. I kept thinking that while I was working on this.

File: src/5e-SRD-Levels.json (hunk 2 above)
Comment (fergcb) [reply src/5e-SRD-Levels.json:8164]: What do you mean, exactly?

File: src/5e-SRD-Levels.json (hunk 3 above)
Comment (fergcb) [reply src/5e-SRD-Levels.json]: I thought I had checked all of them! I'll fix this one and double check.

File: src/5e-SRD-Levels.json (hunk 1 above)
Comment (bagelbits) [reply src/5e-SRD-Levels.json:8182]: Mind cutting another issue! :D

File: src/5e-SRD-Levels.json (hunk 2 above)
Comment (bagelbits) [reply src/5e-SRD-Levels.json:8164]: If we ever have another class with Bonus Proficiency but it's not say "Heavy Armor". Luckily Bard has a similar feature but it's called "Bonus Proficiencies".

Comment (fergcb) [reply src/5e-SRD-Levels.json:8164]: Ah I see. I didn't add any features in this PR, only used existing ones. Perhaps one for a separate issue?

File: src/5e-SRD-Levels.json (hunk 3 above)
Comment (fergcb) [reply src/5e-SRD-Levels.json]: Done! I've written a script to resolve all the reference URLs and they're all coming back positive :)

**Review by bagelbits — APPROVED** [review]
LGTM!

**Comment by ogregoire** [comment]
I wouldn't have merged it yet because the features are all mixed. I'll try to sort them as part of #241

**Comment by fergcb** [comment]
@ogregoire What do you mean, "all the features are mixed"?

**Comment by ogregoire** [comment]
I was under the impression that the order of class features and subclass features was not consistent accross the files. The order is actually consistent, so I don't know why I was so sure of it that I posted this comment. My bad.

**Comment by bagelbits** [comment]
It's okay!

**Comment by fergcb** [comment]
Features weren't touched by this PR, with the exception of renaming Holy Nimbus, anyway. If they aren't/weren't in the right order, that ought to be addressed by a different PR (your general "tidying up" one, perhaps).

**Pull request merged**